locations/spiders/marshalls.py

```diff
@@ -0,0 +1,30 @@
+import scrapy
+import json
+from locations.items import GeojsonPointItem
+
+class MarshallsSpider(scrapy.Spider):
```

Comment:
Can you move this to a separate pull request please?

Reply:
Sorry about that.